types.go

```diff
@@ -597,8 +597,8 @@ type ChosenInlineResult struct {
 // as an inline query result.
 type InputTextMessageContent struct {
 	Text                  string `json:"message_text"`
-	ParseMode             string `json:"parse_mode"`
-	DisableWebPagePreview bool   `json:"disable_web_page_preview"`
+	ParseMode             string `json:"parse_mode,omitempty"`
+	DisableWebPagePreview bool   `json:"disable_web_page_preview,omitempty"`
```

Comment:
Also, none of these types are Marshaled, so I am not sure what the purpose of adding `omitempty` is on them.